specerror/config-linux.go

```diff
@@ -0,0 +1,25 @@
+// Package specerror implements runtime-spec-specific tooling for
+// tracking RFC 2119 violations.
+package specerror
+
+import (
+	"fmt"
+
+	rfc2119 "github.com/opencontainers/runtime-tools/error"
+)
+
+// define error codes
+const (
+	// DefaultFilesystems represents "The following filesystems SHOULD be made available in each container's filesystem:"
+	DefaultFilesystems = "The following filesystems SHOULD be made available in each container's filesystem:"
```

Comment:
Is this really our only `config-linux.md` requirement?  Either way, I'd rather get this PR landed, because the change feels to big to wait until it's complete and perfect.

Reply:
no，these items were already recorded in runtime tools implementation. I add them first. The whole rfc errors will come later.